doc/guide/build.hpp

```diff
@@ -8,6 +8,18 @@ FreeBSD etc). However, mlpack is in the repositories of many Linux distributions
 and so it may be easier to use the package manager for your system.  For example, 
 on Ubuntu, you can install mlpack with the following command:
 
+@code
```

Comment:
I can't directly comment on the line above, anyway, I think it makes sense to be more verbose here, since mlpack-bin will only install the mlpack executables. So I would rephrase it to:

For example, on Ubuntu, you can install the mlpack executables (e.g. mlpack_pca, mlpack_kmeans) with the following command:

Reply:
Yes sure will change this